cypress/README.md

```diff
@@ -43,3 +43,7 @@ docker run --rm  -p 3000:3000 --name=grafana --env GF_PLUGINS_ALLOW_LOADING_UNSI
 ## Troubleshooting
 
 If you run into issues while following these steps, and especially if you already had the plugin installed and built from before, try removing the `node_modules` and `dist` folder and repeating steps 2 and 3. 
+
+There could also be an issue with tests not properly cleaned up, meaning a datasource with the same name cannot be added. You can remove it manually.
```

Comment:
I would rather change the test than documenting this :) you just need to remove the name from the flow

Reply:
🤦‍♀️ I did not realise it was that easy , will do so then!